### ISSUE TITLE
[Bug]: mysql-client hangs during sql execution, need manually ctrl-c

### ISSUE THREAD

**Issue by sukki37** [matrixorigin/matrixone#6330]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 92439b1d3f73552ca912de6180099821c39955fd
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

During running a SQL file of several DDL sqls, the following error occurred on the client side occasionally. All logs can be found here: https://github.com/matrixorigin/matrixone/files/9946003/multi-cn.log.tar.gz
<img width="685" alt="image" src="https://user-images.githubusercontent.com/77312370/200175985-5b58fb31-eb5b-4288-99da-d80ef385d166.png">

### Steps to Reproduce

1. start a cluster with two CNs(If you want to start a mo cluster in eks, you can following this guideline document: https://github.com/matrixorigin/docs/blob/main/dev/guide/MO%20Operator%20%E7%9A%84%E4%BD%BF%E7%94%A8%E4%B8%8E%E6%B5%8B%E8%AF%95%EF%BC%88%E5%86%85%E9%83%A8%E7%89%88%E6%9C%AC%EF%BC%89.md)
2. execute `drop table if exist` `create table xxx` repeatedly

### Additional information

Configuration of each node is here:
[Archive.zip](https://github.com/matrixorigin/matrixone/files/9946010/Archive.zip)

**Comment by daviszhen** [comment]
i am working on it to find the related info.

**Comment by daviszhen** [comment]
I do not reproduce the issue using following eks configuration.
[mo.yaml.zip](https://github.com/matrixorigin/matrixone/files/9949196/mo.yaml.zip)

**Comment by daviszhen** [comment]
I have reproduced in the @sukki37 's eks cluster. I need add debug info and start the cluster outside the eks cluster.

**Comment by daviszhen** [comment]
After syncing with @sukki37 and adding S3 as one feature, I reproduced the case in the eks cluster. The result denoted that the CN service's RPC may take the responsibility. The RPC consumed all the time(10minutes) that came from the context's timeout(10minutes).  
So, The CN people should take over the case.

<img width="1462" alt="hang" src="https://user-images.githubusercontent.com/60595215/200332776-8384f7e3-0ee1-4210-b561-7e6771ed362b.png">

**Comment by fengttt** [comment]
@zhangxu19830126 verified this can be reproed with dev single process, 2cn-1dn setting.

**Comment by nnsgmsone** [comment]
Fixed the problem, but not yet verified. Zhang Xu and Peng Zheng will help verify again.

**Comment by nnsgmsone** [comment]
It should be fixed by #6408. Please ask Peng Zheng to help to cross-check again.

**Comment by fengttt** [comment]
@daviszhen please leave a notes/updates.

**Comment by nnsgmsone** [comment]
I was helping Yuanning check out tpcc #6335. so I didn't look at this issue for now.

**Comment by daviszhen** [comment]
> @daviszhen please leave a notes/updates.

https://github.com/matrixorigin/matrixone/issues/6421 denotes the connection keeps sticking.
But the reason is different from this one.
I have updated the something under https://github.com/matrixorigin/matrixone/issues/6421.

**Comment by JackTan25** [comment]
I'm trying to repro this one, not sure it's solved by someone?

**Comment by JackTan25** [comment]
Now I have Tried to find rpc timeout log, for now, it doesn't appear. Let me continue. My envirment is 2cn,1dn,3 logservice, use eks cluster.

**Comment by JackTan25** [comment]
I tried so many times, but for now, it will result in these. That's the rpc bug, time out. We need to wait zhangxu
[start_up_failed_cn0.txt](https://github.com/matrixorigin/matrixone/files/9985971/start_up_failed_cn0.txt)
[startup_failed_cn1.txt](https://github.com/matrixorigin/matrixone/files/9985972/startup_failed_cn1.txt)

**Comment by aressu1985** [comment]
test for more days , and if not happen, I will closed it

**Comment by aressu1985** [comment]
in progress

**Comment by aressu1985** [comment]
now mutil-cn mo is still not stability, thes types of issues will be regressed till the BVT in mutil-cn mo can pass

**Comment by aressu1985** [comment]
fixed: 0.6.0-1786399（image-tag）
